eng/pipelines/docindex.yml

```diff
@@ -34,16 +35,39 @@ jobs:
           versionSpec: '3.6'
       - template: /eng/common/pipelines/templates/steps/mashup-doc-index.yml
         parameters:
-          SourceDirectory: $(Build.SourcesDirectory)        
-      - task: CopyFiles@2
-        displayName: Copy HTML to Artifacts Directory
-        inputs:
-          sourceFolder: $(Build.SourcesDirectory)/docfx_project/
-          content: '**\*'
-          targetFolder: $(Build.ArtifactStagingDirectory)/docfx_project
-          overWrite: true
+          SourceDirectory: $(Build.ArtifactStagingDirectory)
+      - pwsh: |
+          Copy-Item -Path $(Build.SourcesDirectory)/eng/common/scripts/git-branch-push.ps1 -Destination ./ -Recurse -Force
```

Comment:
We should consider copying all of eng directory incase we get extra dependencies or other tools we use from our normal repos.

Reply:
Do you mean we add a eng/common folder to gh-pages branch?

Reply:
I originally thought eng/common but things like the Language-Settings.ps1 script lives under eng/scripts and so I want to be sure if we start including that in our ps scripts we don't break this. That is why I suggest copying all of the eng directory.